.travis.yml

```diff
@@ -32,6 +33,13 @@ matrix:
           - tox --installpkg ./dist/djangorestframework-*.whl
           - tox  # test sdist
 
+      - python: "3.6"
+        env: TOXENV=dist-optionals
+        script:
+          - python setup.py bdist_wheel
+          - tox --installpkg ./dist/djangorestframework-*.whl
+          - tox  # test sdist
```

Comment:
This env isn't listed in tox?

Reply:
Does not have to be (AFAIK).
Or at least I've assumed that tox-travis will use the explicit env, but haven't checked the build job itself.
(tox itself allows for `tox -e dist-optionals`)